portafly/src/i18n/locales/en/audience/account/account.yml

```
@@ -0,0 +1,60 @@
+---
+# Accounts > Account overview
+
+page_title: Account
+page_title_description: Page title of the subsection for Accounts Overview
+
+# Top area
+
+body_title: Account: {{organization_name}}
```

This requires double quotes to be valid YAML syntax, because of the colon:
```suggestion
body_title: "Account: {{organization_name}}"
```

---

Ah! Okay...